root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -326,6 +335,11 @@ const PART_OF_SERIES_LINK_TYPE_GIDS = _.values(PART_OF_SERIES_LINK_TYPES);
             this.showLinkTypeHelp(!this.showLinkTypeHelp.peek());
         },
 
+        linkTypeName: function () {
```

Comment:
Used `backwards` argument to select the same link phrase in https://github.com/metabrainz/musicbrainz-server/pull/488/commits/8c853861912e9eed2f589c70ff157b63d7993219.